native/core/src/execution/jni_api.rs

```diff
@@ -262,8 +277,11 @@ pub unsafe extern "system" fn Java_org_apache_comet_Native_createPlan(
 fn prepare_datafusion_session_context(
     batch_size: usize,
     memory_pool: Arc<dyn MemoryPool>,
+    local_dirs: Vec<String>,
 ) -> CometResult<SessionContext> {
-    let mut rt_config = RuntimeEnvBuilder::new().with_disk_manager(DiskManagerConfig::NewOs);
+    let disk_manager_config =
+        DiskManagerConfig::NewSpecified(local_dirs.into_iter().map(PathBuf::from).collect());
```

Comment:
Do you know if it is possible to use an object store here?

Reply:
It is not possible. All file operations performed by Spark's [DiskBlockManager]( https://github.com/apache/spark/blob/v4.0.0-rc3/core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala) are using `java.io.File` and `java.nio.file.Files` so it only supports local filesystem. No object store paths should be used for initializing `DiskManagerConfig` here.

Reply:
Ah. Thank you for clarifying this!